assets/src/data/content/utils.tsx

```diff
@@ -127,9 +128,9 @@ export const positionRect = (
     // here we check to see if the view is scrolled past the top of the editor and if so
     // then we adjust the position the toolbar relative to the top of the window so it is
     // always in view
-    const topAdjustedScrollPos = parentRect.top - TOP_SCROLL_OFFSET;
+    const topAdjustedScrollPos = parentRect.top - topScrollOffset;
```

Comment:
Added a bit more padding to the toolbar since it was uncomfortably close to the cursor